Sources/Build/llbuild.swift

```diff
@@ -33,22 +33,30 @@ public struct LLBuildManifestGenerator {
         /// Test target.
         private(set) var test = Target(name: "test")
 
+        /// All targets.
+        var allTargets: [Target] {
+            return [main, test] + otherTargets
+        }
+
         /// All commands.
         private(set) var allCommands = SortedArray<Command>(areInIncreasingOrder: <)
 
+        /// Other targets.
+        private var otherTargets: [Target] = []
```

Comment:
In fact, because the targets are mutable value types, it would be annoying to mutate them if they were stored in a `allTargets` array. I think it is best to keep the `otherTargets` array and keep `allTargets` as a computed property. Do you agree?